vector/src/main/java/im/vector/app/features/settings/devices/v2/list/OtherSessionItem.kt

```diff
@@ -68,6 +72,7 @@ abstract class OtherSessionItem : VectorEpoxyModel<OtherSessionItem.Holder>(R.la
         holder.otherSessionVerificationStatusImageView.render(roomEncryptionTrustLevel)
         holder.otherSessionNameTextView.text = sessionName
         holder.otherSessionDescriptionTextView.text = sessionDescription
+        holder.otherSessionDescriptionTextView.setCompoundDrawablesWithIntrinsicBounds(sessionDescriptionDrawable, null, null, null)
```

Comment:
I am not sure the drawable will have the same tint color as the text. Can you check both in light and dark mode?

Reply:
Fixed in [dc0a36](https://github.com/vector-im/element-android/pull/6965/commits/9dc0a36ca0a66acb3da8ef3676e0110eb5744faa)